distinctipy/color.py

```diff
@@ -0,0 +1,167 @@
+from typing import Tuple, List
+from enum import IntEnum
+from numbers import Number
+
+from .distinctipy import *
+
+
+class ColorOrder(IntEnum):
+    RGB = 1  # Default of the library
+    BGR = 2
+
+
+class DataType(IntEnum):
+    FLOAT = 1  # Default of the library, 0..1 float
+    INT8 = 2  # Used by cv2, 0..255 int
+    INT16 = 3  # 0..65535 int
+
+
+class Color:
+    colors: Tuple[Number, Number, Number]
+
+    _color_order: ColorOrder = ColorOrder.RGB
+    _data_type: DataType = DataType.FLOAT
+
+    def __init__(self, colors: Tuple[Number, Number, Number],
+                 color_order: ColorOrder = ColorOrder.RGB,
+                 data_type: DataType = DataType.FLOAT):
+
+        self.colors = colors
+        self._color_order = color_order
+        self._data_type = data_type
+
+    @classmethod
+    def from_rgb8(rgb8: Tuple[int, int, int]) -> 'Color':
+        return Color(rgb8, color_order=ColorOrder.RGB, data_type=DataType.INT8)
```

Comment:
```suggestion
    @classmethod
    def from_rgb8(cls, rgb8: Tuple[int, int, int]) -> 'Color':
        return cls(rgb8, color_order=ColorOrder.RGB, data_type=DataType.INT8)
```

Thanks @Erol444 ! Just to let you know `@classmethod`s require a first argument `cls` like in this suggestion (it works similar to `self` but it becomes a reference to the class rather than a reference to an instance)

Reply:
oops, my bad! Just updated it.